05-Organizing-and-Executing-Simulations/04-Submitting-Cluster-Jobs.ipynb

```diff
@@ -0,0 +1,611 @@
+{
```

Comment:
<p>"... This displays the generated submission scripts without actually submitting them"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/57/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Done.</p>